image_client/db_utils.py

```diff
@@ -45,41 +48,39 @@ def reset_connection(self):
         self.connect()
 
     def connect(self):
-
         if self.cnx is None:
             self.logger.debug(f"Connecting to db {self.database_host}...")
 
             try:
-                self.cnx = mysql.connector.connect(user=self.database_user,
-                                                   password=self.database_password,
-                                                   host=self.database_host,
-                                                   port=self.database_port,
-                                                   database=self.database_name)
-            except mysql.connector.Error as err:
-                if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+                self.cnx = mariadb.connect(user=self.database_user,
+                                           password=self.database_password,
+                                           host=self.database_host,
+                                           port=self.database_port,
+                                           database=self.database_name)
+            except mariadb.Error as err:
+                print("oh no! error")
```

Comment:
print(f"MariaDB threw an error: {err}")

Reply:
my bad, leftover silly print statement from debugging, although it is a good place to put an error exception, so I'll put in the above print statement.